docs/INTRODUCTION.md

```diff
@@ -1,63 +1,57 @@
+OVE can be used for visual analytics on Large High Resolution Displays, for presentations, or for collaborative group work. It allows a user to control the display of content in web browsers distributed across multiple computers by implementing a microservices architecture that allows the distributed execution of applications using web technologies.
 
+The main components of OVE are [**OVE Core**](https://github.com/ove/ove), which controls sections and the applications running within them and [**OVE Apps**](https://github.com/ove/ove-apps), which provide a set of useful applications for common tasks such as displaying images, displaying webpages, and drawing graphs.
 
-## Concepts
+[**OVE Services**](https://github.com/ove/ove-services) provides core functionality microservices within OVE such as the layout service. [**OVE Asset Services**](https://github.com/ove/ove-asset-services) provides an Asset Manager along with a collection of Asset Processing Services, used to upload archives or divide large networks or images into tiles, and serve these to the corresponding applications.
 
-OVE supports multiple ``spaces``. Each ``space`` is associated with one or more monitors, which may be attached to different computers, that together form a single display. This display may be a Large High Resolution Display (the [Data Observatory](https://www.imperial.ac.uk/data-science/data-observatory/) is 2.53m x 6.00m in size and 30,720 x 4,320 in resolution and consists of 64 monitors), but OVE is also suitable for use on much smaller displays.
+OVE also provides [graphical editors](https://github.com/ove/ove-editor) and [software development kits](https://github.com/ove/ove-sdks) that can be used to design and develop visualisation projects.
 
-In each space, OVE runs within a number of ``clients``. Each ``client`` is a browser window, typically full-screen on a single monitor, but a single ``client`` can span multiple windows if they are attached to the same computer. The arrangement of ``clients`` is described in the [``Spaces.json``](https://github.com/dsi-icl/ove/blob/master/packages/ove-core/src/client/Spaces.json) file.
+## Basic Concepts
 
-``Sections`` are rectangular regions within an ``oveCanvas``. Each ``section`` runs a single OVE ``app``. ``Sections`` may span multiple clients and can overlap.
+An OVE `server` installation supports multiple `spaces`. A `space` is a collection of monitors, which may be attached to different computers, that together form a single display. OVE is designed to be used in Large High Resolution Display environments (such as a [Data Observatory](https://www.imperial.ac.uk/data-science/data-observatory/) that is 2.53m x 6.00m in size and 30,720 x 4,320 in resolution, consisting of 64 monitors). But, OVE is also suitable for use on much smaller displays.
```

Comment:
I think it is useful to communicate that an OVE installation could consist of a single monitor.